dashboard/app/views/plc/tasks/_form.html.haml

```diff
@@ -11,7 +11,7 @@
     = f.text_field :name
     = f.label :learning_module_id
     - options = options_for_plc_task_learning_modules
-    = f.select :plc_learning_module_ids, options_for_select(options, @task.plc_learning_modules.map(&:id)), {}, {multiple: true, size: [options.size, 25].min, style: 'width: 100%'}
+    = learning_modules_for_task_multi_select(f, options, @task.plc_learning_modules)
```

Comment:
hmm - since this is the form for new task generation (it's not used for editing) I really shouldn't be passing anything here. Nothing in the box should be selected.